doc/modules/preprocessing.rst

```diff
@@ -497,6 +497,53 @@ features, one has to explicitly set ``n_values``. For example,
 See :ref:`dict_feature_extraction` for categorical features that are represented
 as a dict, not as integers.
 
+.. _preprocessing_ordinal_features:
+
+Encoding ordinal features
+=============================
+Often categorical features have a clear ordering. For example a person could have features
+``["short", "tall"]``,
+``["low income", "medium income", "high income"]``,
+``["elementary school graduate", "high school graduate", "some college", "college graduate"]``.
+Even though these features can be ordered, we shouldn't necessarily assign scores to them,
+as the difference between categories one and two is not the same as the difference
+between categories two and three.
+
+One possibility to convert these ordinal features to features that can be used
+with scikit-learn estimators is to use a unary encoding, which is
+implemented in :class:`UnaryEncoder`.  This estimator transforms each
+ordinal feature with ``m`` possible values into ``m - 1`` binary features, where the ith
+feature is active if x > i (for i = 0, ... k - 1).
+
+Continuing the example above::
+
+    >>> enc = preprocessing.UnaryEncoder()
+    >>> enc.fit([[0, 0, 3], [1, 1, 0], [0, 2, 1], [1, 0, 2]])  # doctest: +ELLIPSIS
+    UnaryEncoder(dtype=<... 'numpy.float64'>, handle_unknown='error',
+           n_values='auto', ordinal_features='all', sparse=True)
+    >>> enc.transform([[0, 1, 1]]).toarray()
+    array([[ 0.,  1.,  0.,  1.,  0.,  0.]])
+
+By default, how many values each feature can take is inferred automatically from the dataset.
```

Comment:
please limit your lines to under 80 characters

Reply:
done